modules/extension/xsd/xsd-sld/src/test/java/org/geotools/sld/bindings/SLDContrastEnhancementBindingTest.java

```diff
@@ -40,7 +40,7 @@ public void testHistogram() throws Exception {
         assertNotNull(ce.getGammaValue());
         assertEquals(1.23, Filters.asDouble(ce.getGammaValue()), 0d);
 
-        assertNotNull(ce.getType());
-        assertEquals("Histogram", Filters.asString(ce.getType()));
+        assertNotNull(ce.getMethod());
+        assertEquals("histogram", ce.getMethod().name().toLowerCase());
```

Comment:
If you are going to be converting case a lot you may just want to make an additional ContrastMethod method.